tests/test_DataSet.py

```diff
@@ -520,8 +520,7 @@ def test_plot_intenstity_subgroup(self):
         plot_dict = plot.to_plotly_json()
         self.assertEqual(len(plot_dict.get("data")), 3)
 
-    @patch("logging.Logger.warning")
-    def test_plot_intenstity_subgroup_significance_warning(self, mock):
+    def test_plot_intensity_subgroup_gracefully_handle_one_group(self):
```

Comment:
For understanding: we just let it raise the warning instead of mocking it?

Reply:
Should we then assert that the specific warning this tests for is raised, since this was the original intention of self.assertEqual(mock.call_count, 1)?

Reply:
nah, not worth the effort